locales/en/create-kafka-instance-with-sizes.json

```diff
@@ -0,0 +1,110 @@
+    "trial_available_title": "You can create a trial instance to evaluate this service.",
+    "trial_unavailable_message": "All available trial instances are currently in use. Try again later.",
+    "trial_unavailable_title": "$t(common:kafka) instances are unavailable for creation",
+    "trial_used_message": "You can deploy 1 trial instance at a time. To deploy a new instance, delete your existing one first. If you are interested in using this service in production, go to the <0>Streams for Apache Kafka Overview</0> to see what options are available during creation and learn how to contact us.",
+    "trial_used_title": "You already have an instance running"
+  },
+  "no_regions_available": "No regions available",
+  "partitions_value": "up to {{value}}",
+  "quick_start_guide_message": "Need help getting started? Follow our quick start guide.",
+  "quick_start_guide_message_exp": "Follow our quick start guide.",
+  "quick_start_guide_title": "Need help getting started?",
+  "regions_temporarily_unavailable": "Regions temporarily unavailable",
+  "select_cloud_provider": "Select cloud provider",
+  "select_region": "Select region",
+  "size_field_aria": "More info for Size field",
+  "size_help_content": "Size of a Kafka instance is based on streaming units. The number of streaming units defines the capacity, resources, and limits of an instance. An instance with a larger size can handle higher loads and process larger amounts of events, has more storage, and can handle more clients and connections.",
+  "sizes_missing": "Size options display when a cloud provider and region are selected",
+  "some_region_unavailable_helper_text": "One or more regions in the selected cloud provider are temporarily unavailable. Select an available region or try again later.",
+  "standard_kafka_size_description": "Learn how to add streaming units to your account",
+  "standard_kafka_streaming_unit": "Your organization has {{count}} streaming unit remaining",
+  "standard_kafka_streaming_unit_zero": "Your organization has no streaming units remaining",
+  "standard_kafka_streaming_unit_one": "Your organization has {{count}} streaming unit remaining",
+  "standard_kafka_streaming_unit_other": "Your organization has {{count}} streaming units remaining",
+  "statuses": {
+    "creating": "Creating",
+    "degraded": "Degraded",
+    "deleting": "Deleting",
+    "ready": "Ready"
+  },
+  "storage_value": "up to {{value}} GiB",
+  "stream_unit_value": "{{size}} streaming units",
+  "streaming_size_value": "{{value}} streaming unit",
+  "streaming_unit": "streaming units",
+  "temporarily_unavailable": "Temporarily unavailable",
+  "trial_all_region_unavailable_helper_text": "All regions in the selected cloud provider are temporarily unavailable. Select another cloud provider or try again later.",
```

Comment:
Same comment as above...
The original message I provided was too far out in the future, when we have more than one cloud provider. Until we have support for another cloud provider, we shouldn't tell the user to select another cloud provider. But there might also be future situations where we support more than one cloud provider, but don't provide an option.  So maybe we keep this and add another message, or maybe we just update this message for now.

In the case where only one cloud provider is supported, we should say this:
> All regions in the selected cloud provider are temporarily unavailable. Try again later.